#request
string str
---
#response 
triangle[] help_indicators
